<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nCustomPlot</class>
 <widget class="QMainWindow" name="nCustomPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Preferences</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="plotTitle">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fontTitle">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../neutrino-build/src/icons.qrc">
           <normaloff>:/icons/font.png</normaloff>:/icons/font.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Axis</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>430</width>
            <height>142</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="labels_layout" columnstretch="1,0,0,0,0,2">
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="5">
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Range</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Font</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Color</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Log</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Grid</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Label</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="plotName">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveData">
         <property name="toolTip">
          <string>Save Data</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../neutrino-build/src/icons.qrc">
           <normaloff>:/icons/saveTxt.png</normaloff>:/icons/saveTxt.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="copyData">
         <property name="toolTip">
          <string>Copy Data</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../neutrino-build/src/icons.qrc">
           <normaloff>:/icons/saveClipboard.png</normaloff>:/icons/saveClipboard.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadPref"/>
   <addaction name="actionSavePref"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionCopy"/>
   <addaction name="actionExport"/>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../neutrino-build/src/icons.qrc">
     <normaloff>:/icons/saveClipboard.png</normaloff>:/icons/saveClipboard.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy all graphs to clipboard</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../neutrino-build/src/icons.qrc">
     <normaloff>:/icons/saveTxt.png</normaloff>:/icons/saveTxt.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save all graphs to text file</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../neutrino-build/src/icons.qrc">
     <normaloff>:/icons/savePDF.png</normaloff>:/icons/savePDF.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export plot to file</string>
   </property>
  </action>
  <action name="actionLoadPref">
   <property name="icon">
    <iconset resource="../../neutrino-build/src/icons.qrc">
     <normaloff>:/icons/loadPref.png</normaloff>:/icons/loadPref.png</iconset>
   </property>
   <property name="text">
    <string>loadPref</string>
   </property>
   <property name="toolTip">
    <string>Load Preferences</string>
   </property>
  </action>
  <action name="actionSavePref">
   <property name="icon">
    <iconset resource="../../neutrino-build/src/icons.qrc">
     <normaloff>:/icons/savePref.png</normaloff>:/icons/savePref.png</iconset>
   </property>
   <property name="text">
    <string>savePref</string>
   </property>
   <property name="toolTip">
    <string>Save Preferences</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>plotTitle</tabstop>
  <tabstop>fontTitle</tabstop>
  <tabstop>plotName</tabstop>
  <tabstop>saveData</tabstop>
  <tabstop>copyData</tabstop>
 </tabstops>
 <resources>
  <include location="../../neutrino-build/src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
